cstore_fdw.h

```diff
@@ -177,8 +186,10 @@ typedef struct StripeSkipList
  */
 typedef struct ColumnBlockData
 {
-	bool *existsArray;
-	Datum *valueArray;
+	StringInfo existBuffer;
+	StringInfo valueBuffer;
+	CompressionType valueCompressionType;
+	uint32 rowCount;
```

Comment:
We can infer this from ```stripeData->rowCount```, ```blockRowCount```, and ```currentBlockIndex```, and we have only used this in only function, and this is the same value among all columns, so can we remove this from here to make it a bit simpler?